sql/dql/what-is-sql/README.md

```diff
@@ -8,3 +8,6 @@ insights:
   - what-is-a-database-server
   - database-server-types
   - connection-strings
+
+  exercises:
+    - sql-practice-what-is-sql
```

Comment:
This has to match a file name in the folder exactly

Reply:
I created that file: sql/dql/what-is-sql/sql-practice-what-is-sql.md